setup.py

```diff
@@ -45,6 +45,9 @@ def get_readme(name='README.rst'):
 
     # Requirements
     install_requires=['pytango>=9.2.1'],
-    tests_require=['pytest-mock', 'pytest-xdist', 'pytest-coverage'],
+    tests_require=['pytest-mock',
+                   'pytest-xdist',
+                   'pytest-coverage',
+                   'numpy>=1.0.8'],
```

Comment:
It is already a PyTango requirement.
We keep compatibility with numpy verion under 1.8.0 by patching array_equal method (as 1.7.1 is Centos7 default numpy version).
However, unittest requires numpy above 1.8.0 (to test compare_triplet without the patch)